packages/frontend/components/Video/Video.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import ReactPlayer from 'react-player'
+import styles from './videoBoard.module.scss'
+
+export interface VideoInterface {
+  id: string
+  title: string
+  creator: string
+  url: string
+}
+
+interface VideoProps {
+  key: string
+  video: VideoInterface
+}
+
+export default function Video({ video }: VideoProps) {
+  return (
+    <div>
+      <div className={styles.cardContainer}>
+        <div className={styles.descriptor}>
+          <p className={styles.title}>{video.title}</p>
+          <p className={styles.creator}> - {video.creator}</p>
+        </div>
```

Comment:
This should probably eventually scale text with screen size, but for now you can give the css class this:
```
.descriptor{
    overflow: hidden;
    white-space: nowrap;
}
```

Reply:
added